src/table/src/predicate.rs

```diff
@@ -573,6 +573,7 @@ mod tests {
         let file = std::fs::OpenOptions::new()
             .write(true)
             .create(true)
+            .truncate(true)
```

Comment:
Seems unrelated change?

Reply:
We need to truncate here.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/suspicious_open_options

Reply:
Got.